File: src/board/GridHex.java
package board;

public class GridHex extends Grid {

	//Properties
	public final static int[][] DIR = {{-1, 0}, {-1, 1}, {0, 1}, {1, 0}, {1, -1}, {0, -1}};
	
	//Constructors
	public GridHex( int row, int col) {
		
		super( row, col);
		direction = DIR;
	}
	
	//Methods
}

File: Board/GridHex.cs
using System;
namespace Board
{
	public class GridHex : Grid
	{

		//Properties
		public static readonly int[][] DIR = new int[6][]{
			new int[2]{-1, 0},
			new int[2]{-1, 1},
			new int[2]{0, 1},
			new int[2]{1, 0},
			new int[2]{1, -1},
			new int[2]{0, -1}
		};

		public GridHex ( int row, int col) : base(row, col) {

			direction = DIR;
		}

		//Methods
	}
}
